x/ibc/02-client/types/errors.go

```diff
@@ -6,22 +6,22 @@ import (
 
 // IBC client sentinel errors
 var (
-	ErrClientExists                           = sdkerrors.Register(SubModuleName, 1, "light client already exists")
-	ErrClientNotFound                         = sdkerrors.Register(SubModuleName, 2, "light client not found")
-	ErrClientFrozen                           = sdkerrors.Register(SubModuleName, 3, "light client is frozen due to misbehaviour")
-	ErrConsensusStateNotFound                 = sdkerrors.Register(SubModuleName, 4, "consensus state not found")
-	ErrInvalidConsensus                       = sdkerrors.Register(SubModuleName, 5, "invalid consensus state")
-	ErrClientTypeNotFound                     = sdkerrors.Register(SubModuleName, 6, "client type not found")
-	ErrInvalidClientType                      = sdkerrors.Register(SubModuleName, 7, "invalid client type")
-	ErrRootNotFound                           = sdkerrors.Register(SubModuleName, 8, "commitment root not found")
-	ErrInvalidHeader                          = sdkerrors.Register(SubModuleName, 9, "invalid block header")
-	ErrInvalidEvidence                        = sdkerrors.Register(SubModuleName, 10, "invalid light client misbehaviour evidence")
```

Comment:
just curious if there is a reason these errors jump from 10 to 13?

Reply:
Great question!

Reply:
Good catch, probably my bad. There is no reason, can you update them too?

Reply:
yup, done